tasks/drupal.xml

```diff
@@ -255,45 +255,25 @@ Or, you can specify the export file directly:
     <target name="drupal-first-install" depends="set-site">
         <fail unless="drupal.site.admin_user" />
 
+        <symlink link="${build.dir}/${drupal.root}/profiles/contrib/the_build" target="${build.thebuild.dir}/defaults/config/install_profile" />
+
+        <composer command="require" composer="${composer.composer}">
+            <arg value="drupal/config_split" />
+            <arg value="drupal/devel" />
+            <arg value="drupal/workbench" />
+            <arg value="drupal/workbench_tabs" />
```

Comment:
Additional recommendations:
https://www.drupal.org/project/stage_file_proxy
https://www.drupal.org/project/admin_toolbar

I also like to use https://www.drupal.org/project/environment_indicator, but I realize that might not be for everyone.

I think we can hold off on installing any of the more specific functionality like search_api, as that'll prob be on a project-by-project basis.

Reply:
FYI I've created an issue (https://github.com/palantirnet/drupal-skeleton/issues/83) which adds this list of modules into composer.json > suggest.  PR coming soon.

Reply:
@jesconstantine that's cool, but do we have any rationale between why some are pre-installed and some are just suggested? (prob a question for @becw too) but it seems like all of those in suggested could just move to `require`, as they could always be removed too.